src/components/RobotObject.tsx

```diff
@@ -9,7 +9,7 @@ export function RobotObject({ butterClient, onRemove }: { butterClient: HttpClie
 
   useEffect(() => {
     loadAnimations();
-  }, []);
```

Comment:
why removing []?

Reply:
The PR deploy test failed when I had warnings in the code, and this line had a warning for unnecessary [].